tool/gravity/cli/config.go

```diff
@@ -564,7 +564,7 @@ func (i *InstallConfig) validateDNSConfig() error {
 }
 
 func (i *InstallConfig) validateCloudConfig() (err error) {
```

Comment:
I think this call can go to `InstallConfig.NewInstallerConfig` so we have a consistent place for validate-and-set (the role is validated and set in `NewInstallerConfig`) or both need to be in `CheckAndSetDefaults`.

Reply:
Yeah, it's a mess. I'll see if I can unify this somehow.